extensions/wikia/ArticleVideo/bower_components/html5-skin/js/components/accessibilityControls.js

```diff
@@ -6,10 +6,15 @@ var AccessibilityControls = function (controller) {
       "fastForwardRate": 1,
       "lastKeyDownTime": null
     };
-    document.addEventListener("keydown", this.handleKey.bind(this));
+    this.keyEvent = this.handleKey.bind(this);
+    document.addEventListener("keydown", this.keyEvent);
 };
 
 AccessibilityControls.prototype = {
+  cleanUp : function() {
+    document.removeEventListener("keydown", this.keyEvent);
+  },
+
   handleKey: function(e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has a complexity of 14 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)